### PR TITLE
Update contribution-guidelines.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Made minor spacing fix and moved a URL around to open PR and confirm the formatting and header links with author.

@emilyacook, this is awesome, thank you! I opened this PR because:

- I wanted to find out whether the column formatting was intentional before I go in and change it inadvertently. I like the way it looks and can add reference to it in the technical writing style guide if it's how we should style agreements/codes of conduct.

- I wanted to confirm that the first H1 header is meant to go to a GitHub stub here https://github.com/mattermost/.github/blob/master/CONTRIBUTING.md. If so, perhaps we can also add https://mattermost.com/contribute/ as it lists all the areas within which community members can get involved? 

- Alternatively, perhaps we could remove the link as it duplicates the content in that section. 

- Related to that, in the plugin repos we're standardizing the README.md file so that the "Development" section contains the same content and points to two pages that are intended to be the single source of truth (e.g., https://github.com/mattermost/mattermost-plugin-incident-response#contributing). I'm very open to syncing them up for consistency. For example, having the link to https://mattermost.com/contribute/ across the board. 

- I also added what may have been a missing URL but could also have been intentional. 

- Wondering if we can link to this page from here: https://handbook.mattermost.com/contributors/contributors/community-playbook#getting-the-community-involved-in-a-campaign as well? 

Other than my nit-picking, this is super-helpful! :)

### CODE REVIEW

File: contributors/contributors/contribution-guidelines.md [hunk 1]
```diff
@@ -81,19 +82,19 @@ the consequences for any action they deem in violation of this Code of Conduct:
 
 ### 1. Correction
 
-**Community Impact**: Use of inappropriate language or other behavior deemed
+**Community Impact:** Use of inappropriate language or other behavior deemed
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md:85]: Should we specify "inappropriate" or have a link to somewhere that does specify it?

File: contributors/contributors/contribution-guidelines.md [hunk 2]
```diff
@@ -81,19 +82,19 @@ the consequences for any action they deem in violation of this Code of Conduct:
...
 behavior was inappropriate. A public apology may be requested.
 
 ### 2. Warning
 
-**Community Impact**: A violation through a single incident or series
+**Community Impact:** A violation through a single incident or series
 of actions.
 
-**Consequence**: A warning with consequences for continued behavior. No
+**Consequence:** A warning with consequences for continued behavior. No
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
**Consequence:** A private written warning from community leaders with consequences for continued behavior. No
```

File: contributors/contributors/contribution-guidelines.md [hunk 3]
```diff
@@ -81,19 +82,19 @@ the consequences for any action they deem in violation of this Code of Conduct:
...
 interaction with the people involved, including unsolicited interaction with
 those enforcing the Code of Conduct, for a specified period of time. This
 includes avoiding interactions in community spaces as well as external channels
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md:100]:
```suggestion
includes avoiding interactions in community spaces as well as external channels.
```

File: contributors/contributors/contribution-guidelines.md [hunk 4]
```diff
@@ -102,35 +103,33 @@ permanent ban.
...
 with those enforcing the Code of Conduct, is allowed during this period.
 Violating these terms may lead to a permanent ban.
 
 ### 4. Permanent Ban
 
-**Community Impact**: Demonstrating a pattern of violation of community
-standards, including sustained inappropriate behavior,  harassment of an
+**Community Impact:** Demonstrating a pattern of violation of community
+standards, including sustained inappropriate behavior, harassment of an
 individual, or aggression toward or disparagement of classes of individuals.
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
individual, aggression toward, or disparagement of, classes of individuals.
```

File: contributors/contributors/contribution-guidelines.md [hunk 5]
```diff
@@ -102,35 +103,33 @@ permanent ban.
...
 the community.
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+This Code of Conduct is adapted from the [Contributor Covenant][https://www.contributor-covenant.org],
 version 2.0, available at
 https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
 
 Community Impact Guidelines were inspired by [Mozilla's code of conduct
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
Community Impact Guidelines were inspired by [Mozilla's Code of Conduct
```

File: contributors/contributors/contribution-guidelines.md [hunk 6]
```diff
@@ -102,35 +103,33 @@ permanent ban.
...
 enforcement ladder](https://github.com/mozilla/diversity).
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
Enforcement Ladder](https://github.com/mozilla/diversity).
```

File: contributors/contributors/contribution-guidelines.md [hunk 7]
```diff
@@ -102,35 +103,33 @@ permanent ban.
...
 
-[homepage]: https://www.contributor-covenant.org
-
 For answers to common questions about this code of conduct, see the FAQ at
```
Comment (justinegeffen) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
For answers to common questions about this Code of Conduct, see the FAQ at
```

**Comment by emilyacook** [comment]
Thank you so much for looking over this @justinegeffen!

---

> I wanted to find out whether the column formatting was intentional before I go in and change it inadvertently. I like the way it looks and can add reference to it in the technical writing style guide if it's how we should style agreements/codes of conduct.

I think this will probably hit most of your questions, but right now this is purely copy/pasted from those md files (the [contributing guidelines](https://github.com/mattermost/.github/blob/master/CONTRIBUTING.md) one and the [code of conduct](https://github.com/mattermost/.github/blob/master/CODE_OF_CONDUCT.md) one). The reason is that eventually we want the code from those files to be interjected into the handbook automatically so that when the files on the repo are changed, it's reflected back in the handbook (as the files in the repo should be the single source of truth). 

I think since this is the formatting that was chosen for the version in the repo, we should add it to the writing style guide? 

---

> I wanted to confirm that the first H1 header is meant to go to a GitHub stub here https://github.com/mattermost/.github/blob/master/CONTRIBUTING.md. If so, perhaps we can also add https://mattermost.com/contribute/ as it lists all the areas within which community members can get involved?
> Alternatively, perhaps we could remove the link as it duplicates the content in that section.

It is! I debated between having the headers link to it or just having a blurb at the top that's like "To suggest changes, submit a PR in GitHub: (links)" which might be better. I don't think we should completely remove it though.

If we do that then we could definitely also add the contribute link.

---

> Related to that, in the plugin repos we're standardizing the README.md file so that the "Development" section contains the same content and points to two pages that are intended to be the single source of truth (e.g., https://github.com/mattermost/mattermost-plugin-incident-response#contributing). I'm very open to syncing them up for consistency. For example, having the link to https://mattermost.com/contribute/ across the board.

I don't think I have enough context to know what to do for this 😅
CC @jasonblais @aspleenic @hanzei 

---

> I also added what may have been a missing URL but could also have been intentional.

I think that's probably a side-effect of it being copy-pasted from the other files, I can fix that

---

> Wondering if we can link to this page from here: https://handbook.mattermost.com/contributors/contributors/community-playbook#getting-the-community-involved-in-a-campaign as well?

Good idea! I can make that change

**Comment by jasonblais** [comment]
> Related to that, in the plugin repos we're standardizing the README.md file so that the "Development" section contains the same content and points to two pages that are intended to be the single source of truth (e.g., https://github.com/mattermost/mattermost-plugin-incident-response#contributing). I'm very open to syncing them up for consistency. For example, having the link to https://mattermost.com/contribute/ across the board.

Can you help clarify what you mean @justinegeffen?

**Comment by justinegeffen** [comment]
Update on this: Apologies for the delay on this. I'll follow up on this this week.

**Comment by justinegeffen** [comment]
My comment regarding the contribute.md file can be disregarded, @emilyacook and @jasonblais. It looks like everything else has been addressed, thank you!

File: contributors/contributors/contribution-guidelines.md [hunk 8]
```diff
@@ -102,35 +103,33 @@ permanent ban.
...
+individual, aggression toward, or disparagement of, classes of individuals.
 
-**Consequence**: A permanent ban from any sort of public interaction within
+**Consequence:** A permanent ban from any sort of public interaction within
 the community.
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+This Code of Conduct is adapted from the [Contributor Covenant][https://www.contributor-covenant.org],
```
Comment (jasonblais) [inline contributors/contributors/contribution-guidelines.md]:
```suggestion
This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org),
```

**Review by jasonblais — DISMISSED** [review]
Just one formatting fix, else looks good, thanks!

Will defer further reviews to @emilyacook

**Review by jasonblais — APPROVED** [review]

**Comment by justinegeffen** [comment]
@emilyacook, do you think this is good to merge?

**Pull request merged**